devel/libcddb/Portfile

```diff
@@ -1,33 +1,34 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
-
-name             libcddb
-version          1.3.2
-revision         1
-categories       devel
-license          LGPL
-maintainers      nomaintainer
-description      library to access data on a CDDB server
+PortSystem      1.0
+
+name            libcddb
+version         1.3.2
+revision        2
+categories      devel
+license         LGPL
+maintainers     nomaintainer
+description     library to access data on a CDDB server
 long_description \
-	Libcddb is a C library to access data on a CDDB server \
-	(freedb.org). It allows you to search the database for \
-	possible CD matches, retrieve detailed information about \
-	a specific CD, and submit new CD entries to the database.
-homepage         http://libcddb.sourceforge.net/
-platforms        darwin
+    Libcddb is a C library to access data on a CDDB server \
+    (freedb.org). It allows you to search the database for \
+    possible CD matches, retrieve detailed information about \
+    a specific CD, and submit new CD entries to the database.
+homepage        https://libcddb.sourceforge.net
 
-depends_lib      port:libiconv
+depends_lib     port:libiconv
 
-master_sites     sourceforge:project/libcddb/libcddb/${version}
+master_sites    sourceforge:project/libcddb/libcddb/${version}
 
-checksums        md5     8bb4a6f542197e8e9648ae597cd6bc8a \
-                 sha1    2a7855918689692ff5ca3316d078a859d51959ce \
-                 rmd160  e38ed8c7dd5a9b4a5b2a1772a9e98f914872c397
+checksums       rmd160  e38ed8c7dd5a9b4a5b2a1772a9e98f914872c397 \
+                sha256  35ce0ee1741ea38def304ddfe84a958901413aa829698357f0bee5bb8f0a223b \
+                size    352909
 
-use_bzip2        yes
+use_bzip2       yes
 
-configure.args   --without-cdio
+patchfiles      patch-fix-log-headers.diff
```

Comment:
did you submit these changes upstream? If so, please add the URL; if not, please do

Reply:
The last release was in 2009: https://libcddb.sourceforge.net
This is abandoned, but we still need it for dependents.